indico/MaKaC/PDFinterface/conference.py

```diff
@@ -100,26 +100,32 @@ def firstPage(self, c, doc):
         height = self._drawLogo(c)
 
         if not height:
-            height = self._drawWrappedString(c, escape(self._conf.getTitle()), height=self._PAGE_HEIGHT - 2*inch)
+            height = self._drawWrappedString(c, escape(self._event.title.encode('utf-8')),
+                                             height=self._PAGE_HEIGHT - 2*inch)
```

Comment:
I think when mixing +- and */ this can actually improve readability (usually i'd just use parentheses for it though)